src/core/lib/iomgr/executor.cc

```diff
@@ -264,6 +264,16 @@ void Executor::ThreadMain(void* arg) {
     grpc_core::ExecCtx::Get()->InvalidateNow();
     subtract_depth = RunClosures(ts->name, closures);
   }
+
+  /* We have an issue with Apple platforms where applying grp_tls_set here
```

Comment:
No need to use C-style comments.
Also: gpr_tls_set

Reply:
Done